DataHub/Integration/05_ManageLanguages.md

```diff
@@ -0,0 +1,130 @@
+
+#### Procedure
+
+*DataHub > Settings > Tab LANGUAGES*
+
+![Language list](../../Assets/Screenshots/DataHub/Settings/Languages/LanguageList.png "[Language list]")
+
+1. Click the language you want to edit in the list of languages.   
+  The *Edit language* view is displayed.
+
+  ![Edit language](../../Assets/Screenshots/DataHub/Settings/Languages/EditLanguage.png "[Edit language]")
+
+2. Edit the desired data of the language in the corresponding fields.
+
+3. Click the [SAVE] button in the upper right corner.   
+  The changes have been saved. The *Edit language* view is closed.  
+
+
+
+## Deactivate a language
+
+Deactivate a language when it should not be used for a certain time.
+The language is no longer available when it is inactive.
+Further, it is recommended to deactivate a language instead of deleting it to prevent any problems because of existing dependencies.
+
+#### Prerequisites
+
+At least one language has been created, see [Create a language](#create-a-language).
+
+> [Info] By default, the languages *English (United States)* and *German (Germany)* have been predefined in the *Core1 Platform*.
+
+#### Procedure
+
+*DataHub > Settings > Tab LANGUAGES*
+
+![Language list](../../Assets/Screenshots/DataHub/Settings/Languages/LanguageList.png "[Language list]")
+
+1. Click the language you want to edit in the list of languages.   
+  The *Edit language* view is displayed.
+
+  ![Edit language](../../Assets/Screenshots/DataHub/Settings/Languages/EditLanguage.png "[Edit language]")
+
+2. Disable the *Active* toggle.
+
+3. Click the [SAVE] button in the upper right corner.   
+  The language has been deactivated. The *Edit language* view is closed.
+
+
+
+## Delete a language
+
+You can delete a language if it is no longer needed.
+As there are usually existing dependencies, it is highly recommended not to delete a language.
+Instead, you can deactivate a language and thus prevent its use, see [Deactivate a language](#deactivate-a-language).
+
+#### Prerequisites
+
+At least one language has been created, see [Create a language](#create-a-language).
+
+> [Info] By default, the languages *English (United States)* and *German (Germany)* have been predefined in the *Core1 Platform*.
+
+#### Procedure
+
+*DataHub > Settings > Tab LANGUAGES*
+
+![Language list](../../Assets/Screenshots/DataHub/Settings/Languages/LanguageList.png "[Language list]")
+
+1. Select the checkbox of the language you want to delete in the list of languages.    
+  The editing toolbar is displayed above the language list.
+
+2. Click the [Delete] button in the toolbar.  
+  The language has been deleted. The deletion cannot be undone.
+
+  > [Info] If the selected language cannot be deleted, an error message is displayed in the upper right corner. To prevent the languages use, it is recommended to [deactivate the language](#deactivate-a-language) instead of delete it.
```

Comment:
```suggestion
  > [Info] If the selected language cannot be deleted, an error message is displayed in the upper right corner. To prevent the languages use, it is recommended to [deactivate the language](#deactivate-a-language) instead of deleting it.
```